src/screens/Dashboard/CompactTickets.tsx

```diff
@@ -79,6 +79,7 @@ const CompactTickets: React.FC<Props> = ({
         // TODO: Move all this initialization into a better layer of abstraction!
         const travelRights =
           fareContract.travelRights.filter(isPreactivatedTicket);
+        if (travelRights.length < 1) return undefined;
```

Comment:
Maybe this is better as `null`? More common way to return non-existing component

```suggestion
        if (travelRights.length < 1) return null;
```